Makefile

```diff
@@ -36,31 +33,31 @@ ifdef COMPILED_KERNEL_URL
 installer: minimal
 	docker build -t $(IMAGE_NAME):$(VERSION) .
 
-$(DIST)/artifacts/vmlinuz: $(BUILD)/kernel/
+dist/artifacts/vmlinuz: build/kernel/
```

Comment:
Good to ditch `$(DIST)` btw. `dist/`, `build/` and the like is much easier to understand